README.md

```diff
@@ -20,6 +20,7 @@ Start experimenting today:
 * [Getting Started Guide](http://onnx.ai/getting-started)
 * [Supported Frameworks & Tools](http://onnx.ai/supported-tools)
 * [Tutorials on using ONNX converters](https://github.com/onnx/tutorials).
+* [A viewer for ONNX models](https://github.com/lutzroeder/Netron)
```

Comment:
This is a more of top level list of links. Maybe have a separate section below "tools"?

We also have graphviz-based visualizer described here: https://github.com/onnx/tutorials/blob/master/tutorials/VisualizingAModel.md Link it too then?

Reply:
I added 'Tools' and both links, pushed a new commit. Wondering if it makes sense to add this to http://onnx.ai/supported-tools?